components/Event/index.js

```diff
@@ -0,0 +1,96 @@
+import { useState } from "react";
+import EventForm from "../EventForm";
+import { useRouter } from "next/router";
+import { StyledListItem } from "./Event.styled";
+import { formatDate } from "../../resources/dateUtils";
+
+export default function Event({ event, kidsData, mutate }) {
+  const router = useRouter();
+
+  const [isEditMode, setIsEditMode] = useState(false);
+
+  async function handleEditEvent(event) {
+    console.log("Event Object in handleEditEvent:", event);
+    event.preventDefault();
+    const formData = new FormData(event.target);
```

Comment:
Your prop `event` (in line 7) is overshadowed by the local parameter `event` (in line 12). A quick solution is to rename the parameter, see suggestion below.
```suggestion
  async function handleEditEvent(e) {
    console.log("Event Object in handleEditEvent:", e);
    e.preventDefault();
    const formData = new FormData(e.target);
```